examples/objective-c/route_guide/Podfile

```diff
@@ -2,6 +2,8 @@ source 'https://github.com/CocoaPods/Specs.git'
 platform :ios, '8.0'
 
 target 'RouteGuideClient' do
+  pod 'Protobuf', :path => "../../../third_party/protobuf"
```

Comment:
Remember, this means that people will be required to clone submodules before using the examples. Are you sure we want to do that?